app/views/layouts/patient_edit.html.slim

```diff
@@ -2,7 +2,10 @@
   .patient-info
     .row
       .col-md-9
-        h2 = @patient.name
+        h2 
+          = @patient.first_name 
+          | &nbsp;
```

Comment:
What about

= "#{@patient.first_name} #{@patient.last_name}"

to avoid the nbsp?  Or do you really want a non-breaking space?